src/UIAutomation/UiaOperationAbstraction/UiaOperationAbstraction.h

```diff
@@ -973,7 +937,7 @@ namespace UiaOperationAbstraction
                 return std::get<StandinT>(mutableThis.m_member).IsEqual(std::get<StandinT>(mutableRhs.m_member));
             }
 
-            return std::get<ComEnumT>(m_member) == rhs;
+            return std::get<ComEnumT>(m_member) == static_cast<ComEnumT>(rhs);

```

Comment:
>static_cast<ComEnumT>(rhs) [](start = 51, length = 26)

Feels like this is what we usually do??
`std::get<ComEnumT>(rhs.m_member)`

Reply:
That's what the conversion operator is. :) But I am OK with adjusting it.